src/main/kotlin/com/hitc/hitwmapmakerplugin/main/utils/MapFileUtils.kt

```diff
@@ -0,0 +1,28 @@
+package com.hitc.hitwmapmakerplugin.main.utils
+
+import com.hitc.hitwmapmakerplugin.main.core.Map
+import kotlinx.serialization.decodeFromString
+import kotlinx.serialization.json.Json
+import kotlinx.serialization.json.encodeToJsonElement
+import java.io.File
+import java.lang.Exception
+
+
+object MapFileUtils {
+    fun save(map: Map) {
+        val jsonElement = Json.encodeToJsonElement(map)
+        val f = File("./plugins/HitW/test.json")
+        try {
+            f.writeText(jsonElement.toString())
+        }
+        catch (e : Exception) {
+            e.printStackTrace()
+        }
+    }
+
+    fun load(): Map {
+        val f = File("./plugins/HitW/test.json")
```

Comment:
Another thing both files are hard coded to `test.json` adding params to specifie what map / file we are loading / saving is necessary